backend/FwLite/LcmCrdt.Tests/Data/MigrationTests.cs

```diff
@@ -57,6 +57,8 @@ private async Task InitDbFromScripts()
     public async Task GetEntries_WorksAfterMigrationFromScriptedDb()
     {
         await InitDbFromScripts();
+        //manually open the connection, this prevents a race condition where the entries query runs before the collations are created.
+        await _asyncScope.ServiceProvider.GetRequiredService<LcmCrdtDbContext>().Database.OpenConnectionAsync();
```

Comment:
Should we run this after every db commit that might have created a WS?
In this case it was our test db init SQL. But, if a crdt change creates a WS we could have the same problem, right?

Reply:
That's already done in the interceptor on save.

Reply:
Note to self, because I didn't quite understand this response:
We have not only implemented `IDbConnectionInterceptor`, but also `ISaveChangesInterceptor`.